src/sentry/static/sentry/app/data/forms/sentryApplication.jsx

```diff
@@ -1,119 +1,109 @@
 
-const INTERNAL_OPTION_DISABLED_REASON = t(
-  "This option can't be changed once selected. Please make a new integration if you no longer want it to be internal."
-);
-const VERIFY_INSTALL_OPTION_DISABLED_REASON = t(
-  'Internal integrations do not require installation verification.'
-);
-
-const forms = [
+const externalFormFields = [
   {
-    // Form "section"/"panel"
-    title: 'Integration Details',
-    fields: [
-      {
-        name: 'isInternal',
-        label: 'Internal',
-        type: 'boolean',
-        disabled: ({statusDisabled}) => statusDisabled,
-        disabledReason: INTERNAL_OPTION_DISABLED_REASON,
-        help:
-          'If enabled, your integration will automatically be installed and for use within your organization only.',
-      },
-      {
-        name: 'name',
-        type: 'string',
-        required: true,
-        placeholder: 'e.g. My Integration',
-        label: 'Name',
-        help: 'Human readable name of your Integration.',
-      },
-      {
-        name: 'author',
-        type: 'string',
-        required: true,
-        placeholder: 'Acme Software',
-        label: 'Author',
-        help: 'The company or person who built and maintains this Integration.',
-      },
-      {
-        name: 'webhookUrl',
-        type: 'string',
-        required: true,
-        label: 'Webhook URL',
-        placeholder: 'e.g. https://example.com/sentry/webhook/',
-        help:
-          'The URL Sentry will send requests to for events such as installation changes and any Resource Subscriptions the Integration subscribes to.',
-      },
-      {
-        name: 'redirectUrl',
-        type: 'string',
-        label: 'Redirect URL',
-        placeholder: 'e.g. https://example.com/sentry/setup/',
-        help: 'The URL Sentry will redirect users to after installation.',
-      },
-      {
-        name: 'verifyInstall',
-        label: 'Verify Installation',
-        type: 'boolean',
-        disabled: ({changeVerifyDisabled}) => changeVerifyDisabled,
-        disabledReason: VERIFY_INSTALL_OPTION_DISABLED_REASON,
-        help:
-          'If enabled, installations will need to be verified before becoming installed.',
-      },
-      {
-        name: 'isAlertable',
-        type: 'boolean',
-        label: 'Alert Rule Action',
-        help: tct(
-          'If enabled, this integration will be an action under alert rules in Sentry. The notification destination is the Webhook URL specified above. More on actions [learn_more:Here].',
-          {
-            learn_more: (
-              <a href="https://docs.sentry.io/product/notifications/#actions" />
-            ),
-          }
-        ),
-      },
+    name: 'name',
+    type: 'string',
+    required: true,
+    placeholder: 'e.g. My Integration',
+    label: 'Name',
+    help: 'Human readable name of your Integration.',
+  },
+  {
+    name: 'author',
+    type: 'string',
+    required: true,
+    placeholder: 'Acme Software',
```

Comment:
Some of the placeholders start with `e.g.` and some don't. Do we do that on purpose?

Reply:
@markstory Even though this is showing a diff here, none of the actual content in these fields is changing. I am just assigning this list to a stand-alone variable because of the change below. As for starting with `e.g`, I don't know why some do and some don't. Maybe @MeredithAnya or @mnoble know. In either case, I can definitely make a change if we think it makes sense.

Reply:
I think it was just us being not consistent - I'm indifferent to whichever way we wanna do it

Reply:
@MeredithAnya @markstory I'll add the `e.g.` here for consistency